libcloud/compute/drivers/digitalocean.py

```diff
@@ -280,6 +280,40 @@ def ex_power_on_node(self, node):
                                       data=json.dumps(attr), method='POST')
         return res.status == httplib.CREATED
 
+    def ex_rebuild_node(self, node):
+        """
+        Destroy and rebuild the node using its base image.
+
+        :param node: Node to rebuild
+        :type node: :class:`Node`
+
+        :return True if the operation began successfully
+        :rtype ``bool``
+        """
+        attr = {'type': 'rebuild', 'image': node.extra['image']['id']}
+        res = self.connection.request('/v2/droplets/%s/actions' % (node.id),
+                                      data=json.dumps(attr), method='POST')
+        return res.status == httplib.CREATED
+
+    def ex_resize_node(self, node, size_slug):
```

Comment:
[def create_node](https://github.com/adamwight/libcloud/blob/af7de345df569852f258f004606a88973aecd754/libcloud/compute/drivers/digitalocean.py#L130) takes a 'size' object where `size.name` is the size slug. What do you think about also passing `size` here instead of `size_slug` to keep the methods uniform?